pkg/cmd/server/bootstrappolicy/securitycontextconstraints_test.go

```diff
@@ -25,11 +27,20 @@ func TestBootstrappedConstraints(t *testing.T) {
 	groups, users := GetBoostrapSCCAccess(DefaultOpenShiftInfraNamespace)
 	bootstrappedConstraints := GetBootstrapSecurityContextConstraints(groups, users)
 
-	if len(expectedConstraints) != len(bootstrappedConstraints) {
-		t.Errorf("unexpected number of constraints: found %d, wanted %d", len(bootstrappedConstraints), len(expectedConstraints))
+	if len(expectedConstraintNames) != len(bootstrappedConstraints) {
+		t.Errorf("unexpected number of constraints: found %d, wanted %d", len(bootstrappedConstraints), len(expectedConstraintNames))
 	}
 
+	var orderedBootstrappedConstraints []*securityapi.SecurityContextConstraints
```

Comment:
Why do we need to introduce `orderedBootstrappedConstraints`? Why we can't sort `bootstrappedConstraints` and use it later?